File: discover_ruby/write_them_to_disc.rb
require 'httpclient'
require 'uri'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 1ce0fbaacb310b39ccf742a79fd0de2238a95b8a'
}

client=HTTPClient.new
uri=URI.parse('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc')
result= client.get(uri,nil,extheaders)
File.open('/tmp/37', 'w') do  |file| 
  file.puts result.content
end
puts 'The file was saved!'

File: write_them_to_disc.py
import urllib
import urllib2
import json

request_headers = {
    'User-Agent': 'Holberton_School',
    'Authorization': 'token 1dcf26a7307f01b81955cc135df2790a61527f91'
}

url='https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc'

req=urllib2.Request(url,headers=request_headers)
res=urllib2.urlopen(req)

data=res.read()
f= open('/tmp/37','w')
f.write(data)
print 'The file was saved!'
